### PR TITLE
feat(components): use lapis 2

### LINKED ISSUE #54: Migrate to LAPIS 2
Currently, the stories use the open LAPIS 1 instance. Migrate to LAPIS 2.

### PR DESCRIPTION
**Pull request by JonasKellerer**
resolves #54

### Summary

Use LAPIS2 for queries
  - use zod to parse requests
  - errors from LAPIS will be directly thrown
  - use POST versions of endpoints instead of GET (makes it easier to query)

### PR Checklist

~~- [ ] The implemented feature is covered by an appropriate test.~~

### CODE REVIEW

File: components/src/preact/prevalenceOverTime/prevalence-over-time.stories.tsx
Comment (fengelniederhammer) [inline components/src/preact/prevalenceOverTime/prevalence-over-time.stories.tsx:1]: This story still sends LAPIS call, probably due to `fields: 'date',`

File: components/src/preact/relativeGrowthAdvantage/relative-growth-advantage.stories.tsx
Comment (fengelniederhammer) [inline components/src/preact/relativeGrowthAdvantage/relative-growth-advantage.stories.tsx:1]: This also sends LAPIS calls.

File: components/src/web-components/display/relative-growth-advantage-component.stories.ts
Comment (fengelniederhammer) [inline components/src/web-components/display/relative-growth-advantage-component.stories.ts:1]: This also sends LAPIS calls.

**Review by fengelniederhammer — APPROVED** [review]
LGTM

File: components/src/lapisApi/lapisTypes.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    mutation: z.string(),
+    proportion: z.number(),
+    count: z.number(),
+});
+export const mutationsResponse = makeLapisResponse(z.array(mutationProportionCount));
+
+const insertionCount = z.object({
+    insertion: z.string(),
+    count: z.number(),
+});
```
[the remark this answers is not in the document]
Comment (JonasKellerer) [reply components/src/lapisApi/lapisTypes.ts]: I now included the new fields of the endpoints. They are currently not used and waiting for #123

**Pull request merged**